### ISSUE TITLE
Patcher Request - Reach Tree Remover

### ISSUE THREAD

**Issue by ex0-tekk** [Synthesis-Collective/Development#18]
I cannot find a single setup where Reach Trees look good in VR, they are shimmering eye sores close up & at a distance they stick out as pale white vertical sticks, imo they are hands down the most unsightly trees in Skyrim and look completely out of place, they are also incredibly visually distracting in VR because of these issues, no one seems to have ever made a Reach Tree Remover Mod & I have found No LOD setting or Mod that fixes this unsightly tree, I am not proficient with CK to make mods & I would love to see a simple Reach Tree Remover Synth patcher developed that plays nice with DynDOLOD. (If that's even a concern for a patcher of this kind.)

**Comment by mrudat** [comment]
Is it a specific kind of tree (or set of trees) that only appears in The Reach, or is it trees in The Reach in general that don't look good?

Collecting a few IDs from in-game trees would probably help if it's a tree or kind of tree that offends.

Do you just want them gone, or would replacing them with different trees also be an option?

https://www.nexusmods.com/skyrimspecialedition/mods/24563 <-- do you know if this mod happens to identify all or just some of the trees in question?

**Comment by ex0-tekk** [comment]
I don't think that mod identifies all of them, those **are** the trees however, and the majority of them do not have rocks under them..... but yes, the twiggy bare branched trees in the Tundra / Reach and inside / outside Markarth, I'll try to collect some IDs , and I'd prefer them all 100% removed if possible, I recently noticed some of these trees also have small patches of hanging moss on the branches, not sure if this is a separate ID.

Update: It can be very difficult to get accurate ID numbers in SkyrimVR because the desktop screen resolution is skewed and the mouse cursor seems to pick up on multiple numbers even when the user is most definitely clicking the proper surface area to attain said ID.... I did my best. 

0001ed2b
0006e57e
00052b1c
00088643
000885fc
00088607
0006e597
a0004db0
000b8c67
0006d2b8
0006edee

I took the IDs of a few trees from inside Markarth, outside Markarth and in the tundra near Whiterun....i hope these are right, it seems every tree has a diff set of numbers? ... I was getting multiple ID results on each tree. (this could be a result of VR desktop screen cursor inaccuracy, or maybe these tree ID's come in multiples?)

**Comment by ex0-tekk** [comment]
Can a reach tree remover synth patch be made from this mod so others can benefit without needing this mod ? any permissions needed from me etc ?  DO IT! lol
a friend of mine made the mod but i am hosting it on nexus and i have full permission to use the mod as i wish.
https://www.nexusmods.com/skyrimspecialedition/mods/52347
